practice/004/ex.ml

```diff
@@ -0,0 +1,16 @@
+
+open OUnit2
+
+module type Testable = sig
+  val length : 'a list -> int
+end
+
+module Make(Tested: Testable) : sig val v : test end = struct
+  let v = "length" >::: [
+    "nil" >:: (fun _ -> assert_equal (Tested.length []));
+    "cons" >:: (fun _ -> assert_equal (Tested.length ["a"; "b"; "c"]));
```

Comment:
You must specify the expected return value in both cases here

Reply:
Thank you, on it.